cime_config/acme/machines/Makefile

```diff
@@ -185,14 +185,40 @@ ifndef AR
    AR := ar
 endif
 
-ifdef NETCDF_PATH
-  ifndef INC_NETCDF
-    INC_NETCDF:=$(NETCDF_PATH)/include
-  endif
-  ifndef LIB_NETCDF
-    LIB_NETCDF:=$(NETCDF_PATH)/lib
+# To prevent breaking cleaning and building components which don't require NetCDF directly
+ifeq ($(if $(or $(filter pio, $(subst /,  , $(MAKECMDGOALS))), $(filter exec_se, $(subst /,  , $(MAKECMDGOALS)))), true, false), true)
```

Comment:
Can you explain what you are doing here - I'm sure we can find a less complicated method.